pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -50,10 +50,28 @@ msgctxt "#30007"
 msgid "Response timeout in seconds"
 msgstr ""
 
-#empty strings from id 30008 to 30099
+msgctxt "#30008"
+msgid "Advanced settings"
+msgstr ""
+
+#empty strings from id 30009 to 30049
+
+msgctxt "#30050"
+msgid "Auto recordings"
+msgstr ""
+
+msgctxt "#30051"
+msgid "Use approximate start time instead of fixed start time"
```

Comment:
Yep, gettin rid of spinners is much appreciated.  There are quite some more in PVR-related dialogs which can get replaced with better solutions. (timer and search dialog for example if I´m not mistaken)
In general the replacement options are:
- radiobuttons (for 2-state)
- DialogSelect
- button controls with a toggling label2